webextensions/_locales/ja/messages.json

```
@@ -206,16 +206,16 @@
   "config_userRule_confirmMessage_caption_subjectOrBody":    { "message": "警告のメッセージ (\"%S\" と書いた位置に語句が入ります):" },
 
 
-  "config_attentionDomains_caption": { "message": "特別に注意が必要な外部ドメインの宛先" },
-  "config_blockedDomains_caption": { "message": "送信を禁止する宛先" },
+  "config_attentionDomains_caption": { "message": "特別に注意が必要な外部ドメイン・アドレスの宛先" },
```

```suggestion
  "config_attentionDomains_caption": { "message": "特別に注意が必要な外部ドメイン・アドレス" },
```

「アドレス」と「宛先」は同じ意味なので、「アドレスの宛先」は違和感がありました。
単に列挙して意図は伝わるのではないかと思いました。

---

アドレスはaddress、宛先はrecipientの訳なのですが、

* recipient "YUKI Hiroshi <yuki@clear-code.com>"
  * name "YUKI Hiroshi"
  * addres "yuki@clear-code.com"
    * local part "yuki"
    * domain part "clear-code.com"

という構造に依拠しての説明なので、同じ意味というわけでは実はないのです……
（ただ、「一般利用者視点で同じ物を指しているように見える」のは事実なので、わかりにくい表現という指摘は妥当と思います）

---

Thunderbirdではrecipientを「宛先」で翻訳する傾向にありますか？
recipientを「受信者」で翻訳すれば誤解しにくい気もしまして...。

---

※宛先のままでも異論ありません。

---

recipientは状況によって送信者、受信者いずれにもなるので「宛先」という翻訳になっていることも納得できる。
（FlexConfirmMailは送信先のチェックのアドインなので、UI上は受信者の意味でしか使われなさそうだが。）

---

いや、「宛先」だと受信者、Cc、Bccを指すことができて、むしろこの方が良いのか？

---

いや、でも新規メール作成時などの送信先は「宛先」か。

---

ThunderbirdのUI上では、メール作成ウィンドウだと「宛先」、受信済みメールの一覧のカラムでは「受信者」、受信済みメールの個別表示のヘッダー領域では「宛先」になっていますね。訳がまちまち……

---

ともかく、傾向的には「宛先」のほうをよく見かける印象があります。

---

なるほど。ありがとうございます。
とりあえず、Thunderbirdでも宛先を使っていたりしますし、取り下げます。